analyticsclient/_base.py

```diff
@@ -0,0 +1,43 @@
+import analyticsclient.constants.data_format as DF
```

Comment:
Nit: `DF` is not terribly descriptive, can you name this `DATA_FORMAT` or something similar?

Question: why does this file contain a leading `_`? That feels unnecessary.